deployment-aws/terraform/cloudtrail.tf

```diff
@@ -1,11 +1,11 @@
-resource "aws_cloudtrail" "webmev-cloudtrail" {
-  name                    = "${local.common_tags.Name}"
-  s3_bucket_name          = aws_s3_bucket.log_bucket.id
-  s3_key_prefix           = "${local.stack}"
-  include_global_service_events = false
+resource "aws_cloudtrail" "webmev_cloudtrail" {
```

Comment:
Referencing `aws_cloudtrail.webmev-cloudtrail` is redundant, so I'd use `main` or `default`, for example: https://github.com/web-mev/mev-backend/blob/dev/deployment-aws/terraform/database.tf#L5

Also, indentation needs fixing in this resource: https://developer.hashicorp.com/terraform/language/syntax/style